client/src/components/Navbar/useStyles.ts

```diff
@@ -0,0 +1,72 @@
+import { makeStyles } from '@mui/styles';
+import { Theme } from '@mui/material/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    backgroundColor: 'white !important',
+  },
+  toolbar: {
+    boxShadow: '1px 1px 5px lightgrey, -1px -1px 5px lightgrey',
+    height: '10vh',
+  },
+  login: {
+    width: 120,
+    height: 45,
+    borderRadius: theme.shape.borderRadius,
+    fontSize: 16,
+    backgroundColor: 'white !important',
+    fontWeight: 'bold',
+    color: '#f14140',
+    border: 'solid 1px #f14140 !important',
+    marginRight: '10px !important',
+  },
+  signup: {
+    width: 120,
+    height: 45,
+    borderRadius: theme.shape.borderRadius,
+    fontSize: 16,
+    backgroundColor: '#f14140',
+    fontWeight: 'bold',
+    marginRight: '25px !important',
```

Comment:
`important` ignores CSS specificity, and acts as a directive to always use that style. Resort to using the `sx` prop in MUI instead